pkg/plugins/backendplugin/secretsmanagerplugin/secretsmanager.proto

```diff
@@ -3,54 +3,62 @@ package secretsmanagerplugin;
 
 option go_package = "./;secretsmanagerplugin";
 
-message SecretsGetRequest {
+message Key {
+    int64 orgId = 1;
+    string namespace = 2;
+    string type = 3;
+}
+
+message GetSecretRequest {
     Key keyDescriptor = 1;
 }
 
-message SecretsSetRequest {
+message GetSecretResponse {
+    string userFriendlyError = 1;
+    string decryptedValue = 2;
+    bool exists = 3;  
+}
+
+message SetSecretRequest {
     Key keyDescriptor = 1;
     string value = 2;
 }
 
-message SecretsDelRequest {
-    Key keyDescriptor = 1;
+message SetSecretResponse {
+    string userFriendlyError = 1;
 }
 
-message SecretsKeysRequest {
+message DelSecretRequest {
     Key keyDescriptor = 1;
-    bool allOrganizations = 2;
 }
 
-message SecretsRenameRequest {
-    Key keyDescriptor = 1;
-    string newNamespace = 2;
+message DelSecretResponse {
+    string userFriendlyError = 1;
 }
 
-message Key {
-    int64 orgId = 1;
-    string namespace = 2;
-    string type = 3;
+message ListSecretsRequest {
+    Key keyDescriptor = 1;
+    bool allOrganizations = 2;
 }
 
-message SecretsErrorResponse {
-    string error = 1;
+message ListSecretsResponse {
+    string userFriendlyError = 1;
+    repeated Key keys = 2;
 }
 
-message SecretsGetResponse {
-    string error = 1;
-    string decryptedValue = 2;
-    bool exists = 3;  
+message RenameSecretRequest {
+    Key keyDescriptor = 1;
+    string newNamespace = 2;
 }
 
-message SecretsKeysResponse {
-    string error = 1;
-    repeated Key keys = 2;
+message RenameSecretResponse {
+    string userFriendlyError = 1;
 }
 
-service RemoteSecretsManager {
-    rpc Get(SecretsGetRequest) returns (SecretsGetResponse);
-    rpc Set(SecretsSetRequest) returns (SecretsErrorResponse);
-    rpc Del(SecretsDelRequest) returns (SecretsErrorResponse);
-    rpc Keys(SecretsKeysRequest) returns (SecretsKeysResponse);
-    rpc Rename(SecretsRenameRequest) returns (SecretsErrorResponse);
+service SecretsManager {
+    rpc Get(GetSecretRequest) returns (GetSecretResponse);
+    rpc Set(SetSecretRequest) returns (SetSecretResponse);
+    rpc Del(DelSecretRequest) returns (DelSecretResponse);
+    rpc Keys(ListSecretsRequest) returns (ListSecretsResponse);
+    rpc Rename(RenameSecretRequest) returns (RenameSecretResponse);
```

Comment:
What do you think of these suggestions?

```suggestion
    rpc GetSecret(GetSecretRequest) returns (GetSecretResponse);
    rpc SetSecret(SetSecretRequest) returns (SetSecretResponse);
    rpc DeleteSecret(DeleteSecretRequest) returns (DeleteSecretResponse);
    rpc ListSecrets(ListSecretsRequest) returns (ListSecretsResponse);
    rpc RenameSecret(RenameSecretRequest) returns (RenameSecretResponse);
```

Reply:
I'm a big fan of descriptive variable and function names, so I like these. I will make the changes and merge. Thanks!